client/src/components/mapping/ConceptMap/index.tsx

```diff
@@ -348,7 +348,9 @@ const ConceptMapDialog = ({
             className="right"
             onChange={(e: React.FormEvent<HTMLElement>): void => {
               const target = e.target as HTMLInputElement;
-              setConceptMapTitle(target.value);
+              if (/^\S*$/.test(target.value)) {
```

Comment:
do not set the concept map title if it matches the regex ?

Reply:
It's on each keystroke, so it just doesn't doesn't add a space if the user tries to add one